### DISCUSSION TITLE
Nesting Prefabs

### DISCUSSION THREAD

**Discussion by ncomendant** [rewin123/space_editor#142] [Q&A] [answered]
From the readme:

![image](https://github.com/rewin123/space_editor/assets/3222456/e2c36977-4959-4ec8-8c3a-9f856c5a94d6)

I looked through the examples, but could not find an example of this. How do I nest a prefab into another?

**Comment by naomijub** [comment]
Probably @rewin123 can explain this better, but I have added an issue to address this https://github.com/rewin123/space_editor/issues/144

**Comment by rewin123** [comment, marked as answer]
![image](https://github.com/rewin123/space_editor/assets/10954133/5e14d334-ceaf-44a0-8d7e-4820baecb9ca)

Thanks for the question! 

Prefabs can contain other prefabs inside them, defined by the path specified in the PrefabLoader component. The picture is an example of how this works, I created an empty prefab in the editor, added one entity with the PrefabLoader component and specified the path to an existing saved prefab.

I will add example in 0.4 version

**Reply by ncomendant** [reply]
I totally missed that. That's awesome!

Thanks for the help. This is an awesome project and I really appreciate all of the effort that went into this.
